tests/Microsoft.Identity.Test.ConfigurationProvider/CloudConfigurationProvider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Specialized;
+using System.IO;
+//using System.Configuration;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Identity.Test.ConfigurationProvider
+{
+    public static class CloudConfigurationProvider
+    {
+        const string CloudSelectionConfig = "cloudSelection.json";
```

Comment:
How are the builds going to deploy this json file? It will complicate our build definitions...

Reply:
removed json file